macaroonbakery/discharge.py

```diff
@@ -76,18 +81,19 @@ def check_third_party_caveat(self, ctx, info):
         :param ctx (AuthContext)
```

Comment:
If the caveat kind was not recognised, the checker should
    raise a CaveatNotRecognized exception; if the check failed,
    it should raise a ThirdPartyCaveatCheckFailed exception.


?

Reply:
done.